blazor/scheduler/appointments.md

```diff
@@ -477,6 +477,63 @@ By default, the scheduler will render the overlapping events based on the start
 ```
 ![Blazor Scheduler with Overlapping Events](images/blazor-scheduler-overlapping-events.png)
 
+## Preventing Overlapping Events
+
+In industries like healthcare and legal services, avoiding overlapping appointments is crucial for compliance and confidentiality. Setting the `AllowOverlap` property to false prevents conflicts and triggers an alert if an overlap is attempted.
+
+>Note: By default [AllowOverlap](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Schedule.SfSchedule-1.html#Syncfusion_Blazor_Schedule_SfSchedule_1_AllowOverlap) is true. You can obtain details of overlapping appointments via the `args` parameter in the [PopupOpenEventArgs](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Schedule.PopupOpenEventArgs-1.html).
+
+```cshtml
+@using Syncfusion.Blazor.Schedule
+
+<SfSchedule TValue="AppointmentData" AllowOverlap="false" Height="550px" @bind-SelectedDate="@CurrentDate">
+    <ScheduleEventSettings DataSource="@DataSource"></ScheduleEventSettings>
+    <ScheduleEvents TValue="AppointmentData" OnPopupOpen="OnPopupOpen"></ScheduleEvents>
+    <ScheduleViews>
+        <ScheduleView Option="View.Day"></ScheduleView>
+        <ScheduleView Option="View.Week"></ScheduleView>
+        <ScheduleView Option="View.WorkWeek"></ScheduleView>
+        <ScheduleView Option="View.Month"></ScheduleView>
+        <ScheduleView Option="View.Agenda"></ScheduleView>
+    </ScheduleViews>
+</SfSchedule>
+
+@code{
+    DateTime CurrentDate = new DateTime(2020, 1, 31);
+    List<AppointmentData> DataSource = new List<AppointmentData>
+    {
+        new AppointmentData { Id = 1, Subject = "Meeting", StartTime = new DateTime(2020, 1, 31, 9, 30, 0), 
+        EndTime = new DateTime(2020, 1, 31, 11, 0, 0) },
+        new AppointmentData { Id = 2, Subject = "Conference", StartTime = new DateTime(2020, 1, 29, 10, 0, 0), 
+        EndTime = new DateTime(2020, 1, 29, 11, 0, 0) },
+        new AppointmentData { Id = 3, Subject = "Tech discussion", StartTime = new DateTime(2020, 1, 29, 10, 30, 0), 
+        EndTime = new DateTime(2020, 1, 29, 11, 30, 0) },
+        new AppointmentData { Id = 4, Subject = "London", StartTime = new DateTime(2020, 1, 26, 12, 0, 0), 
+        EndTime = new DateTime(2020, 1, 26, 13, 0, 0) },
+        new AppointmentData { Id = 5, Subject = "New York", StartTime = new DateTime(2020, 1, 25, 13, 0, 0), 
+        EndTime = new DateTime(2020, 1, 25, 15, 0, 0) },
+    };
+    public void OnPopupOpen(PopupOpenEventArgs<AppointmentData> args)
+    {
+        List<AppointmentData> overlappingData = args.OverlapCollection;
+    }
+    public class AppointmentData
+    {
+        public int Id { get; set; }
+        public string Subject { get; set; }
+        public DateTime StartTime { get; set; }
+        public DateTime EndTime { get; set; }
+    }
+}
+```
+### Limitations
```

Comment:
Limitations:

- During the initial loading, conflicting events are not rendered on the scheduler. For conflicting recurrence series, only non-conflicting occurrences are displayed.
- During dynamic interactions such as adding, editing, dragging, or resizing events, the scheduler checks for conflicts. If overlaps occur, actions are prevented, and users are alerted. Conflicts in recurrence series block all occurrences.